### DISCUSSION TITLE
Using cypress to login through email

### DISCUSSION THREAD

**Discussion by ramiel** [nextauthjs/next-auth#826] [Help]
**Your question**
I'm trying to use cypress to login through email, up to the point when the email is sent. The test is this:

```js
  it.only('can login with an email', () => {
    cy.visit('/');
    cy.get('input[name="email"]').focus().type(testEmail);
    cy.get('form').submit();
    cy.url().should('include', '/auth/verify-request');
  });
```

the form gest compiled and submitted but the page returns to `/auth/signin`. Do you know what's the problem? The form is working fine manually.

**Feedback**
*Documentation refers to searching through [online documentation](https://next-auth.js.org), code comments and issue history. The example project refers to [next-auth-example](https://github.com/iaincollins/next-auth-example).*

* [ ] Found the documentation helpful
* [ ] Found documentation but was incomplete
* [x] Could not find relevant documentation
* [ ] Found the example project helpful
* [ ] Did not find the example project helpful

**Comment by TomGroombridge** [comment]
I'm having the same problem. Could someone please show how to use Cypress for passwordless
